test/fixtures/schema/string-literal.json

```diff
@@ -164,7 +164,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"street\": [\n    \"bye bye\",\n    \"san francisco\"\n  ]\n}"
+                              "content": "{\n  \"street\": [\n    \"\",\n    \"bye bye\",\n    \"san francisco\"\n  ]\n}"
```

Comment:
This is an interesting one:

```apib
# Group Example
# Resource [/example]

## Action [GET]

+ Response 200 (application/json)

    + Attributes
        - street (array[string], fixed)
            - bye bye
            - san francisco
```

The empty `string` which the user intended as a "type hint" as `array[string]` is becoming a fixed value in the output. The current JSON Schema is adding it two. So the matching JSON Schema requires this change to validate properly.

HOWEVER, the JSON Schema and example in Drafter 3.2.7 stable doesn't do this.

Reply:
That's because the JSON Schema and JSON Value renderers get some API Elements, and they cannot know that the API Blueprint was what it was. Check out the corresponding API Elements:https://github.com/apiaryio/drafter/blob/f17705e4b74a134db5848bf3d40cbc4eebb07f12/test/fixtures/schema/string-literal.json#L129

Reply:
So, this might be a bug, but it's a bug during AST -> API Elements transformation, not API Elements -> JSON Schema/Value.